cerebro.spec.in

```diff
@@ -310,65 +198,30 @@ fi
 %{_libdir}/cerebro/*cerebro_metric_txerrs.*
 %endif
 
-%post metric-slurm-state
-#
-# Restart cerebrod if already running
-#
-if [ "$1" = 1 ]; then
-   if [ -x %{_initrddir}/cerebrod ]; then
-      [ -x /sbin/chkconfig ] && /sbin/chkconfig --add cerebrod
-   fi
-fi
-if [ $1 -ge 1 ]; then
-   if [ -x %{_initrddir}/cerebrod ]; then
-      %{_initrddir}/cerebrod condrestart
-   fi
-fi
-
 %if %{?_with_slurm_state:1}%{!?_with_slurm_state:0}
 %files metric-slurm-state
 %defattr(700,root,root)
 %{_libdir}/cerebro/*cerebro_metric_slurm_state.*
 %endif
 
-%post metric-shutdown
-#
-# Restart cerebrod if already running
-#
-if [ "$1" = 1 ]; then
-   if [ -x %{_initrddir}/cerebrod ]; then
-      [ -x /sbin/chkconfig ] && /sbin/chkconfig --add cerebrod
-   fi
-fi
-if [ $1 -ge 1 ]; then
-   if [ -x %{_initrddir}/cerebrod ]; then
-      %{_initrddir}/cerebrod condrestart
-   fi
-fi
-
 %if %{?_with_shutdown:1}%{!?_with_shutdown:0}
 %files metric-shutdown
 %defattr(700,root,root)
 %{_libdir}/cerebro/*cerebro_metric_shutdown.*
 %endif
 
-%post event-updown
-#
-# Restart cerebrod if already running
-#
-if [ "$1" = 1 ]; then
-   if [ -x %{_initrddir}/cerebrod ]; then
-      [ -x /sbin/chkconfig ] && /sbin/chkconfig --add cerebrod
-   fi
-fi
-if [ $1 -ge 1 ]; then
-   if [ -x %{_initrddir}/cerebrod ]; then
-      %{_initrddir}/cerebrod condrestart
-   fi
-fi
-
```

Comment:
do we need the systemd equivalents for all the modules?  Otherwise there could be racing on which modules get installed first/last?